crossteam-project-valuation.md

```diff
@@ -0,0 +1,101 @@
+# Title
+
+Cross-team Project Valuation
+
+# Patlet
+
+It's hard to sell the value of cross-team, inner sourced projects that don't provide a direct impact on company revenue.
+Here's a data-driven way to represent your project that both articulates its value and amplifies it.
+
+# Context
+
+* You are responsible for a cross-team that serves as a platform for others in the company.
+* The cross-team project does not deliver any direct value for company revenue.
+
+# Problem
+
+Cross-team projects can potentially have a very large impact on the company yet are notoriosly-difficult to represent in a data-driven fashion.
+As a result, it is easy and common to either pursue projects that does not provide real value or to underfund what would otherwise produce great value.
+Setting in place a pattern and model of how to value such projects gives us the tool that we need to focus and amplify high-value collaboration for the company.
+
+# Forces
+
+Projects need to show value (objective or subjective) to company leadership in order to be funded.
+
+# Solution
+
+The core of all cross-team project value is the idea that we can get more done together than apart.
+Ascribing value to a cross-team effort is an exercise in quantifying _how much more_ is getting done together.
+The exact delta in productivity will vary by domain and project.
+There is a common process, though, by which you can create a model to calculate it.
+
+## Explanation
+
+Assemble a small team of subject matter experts in your domain.
+Using that team of experts, estimate 4 things about each consumer of your project output:
+  * How long does it take them to consume your project output?
+  * How long would it otherwise take them to home-roll the value of your project output for themselves?
+  * What percentage of your project output is actually useful for them?
+  * How much time on an ongoing basis (ideally per-use) would they otherwise spend maintaining their home-rolled solution?
+
+When making these estimations, it's impossible to know with high precision _exactly_ how long any activities take.  That's not your goal.
+Rather than exactness, you should strive to _**set a worst-case bound**_ on these estimates.
+The idea is for the group of experts to be able to say to each other, "We don't know exactly how long it would take, but we can all agree it's _at least_ this much."
+Specifically, you should estimate a *maximum* reasonable time to consume your project output and *minimum* reasonable times for consumers to otherwise home-roll, use and maintain their own solutions.
+
+> One note about cost-to home-roll.  The cost to home-roll a solution is NOT necessarily (very unlikely, in fact) the same as the cost of making a shared solution.
+Oftentimes for the same functionality the modularity and quality involved in buliding a cross-team, shared solution makes it a noticeably higher investment than a quick, hardcoded implementation used just once.
```

Comment:
Indeed. If I remember right from my software product line times, the rule of thumb was to not make something reusable unless it was used at least three times.

Reply:
@rrrutledge looks like this conversation can be resolved?